### ISSUE TITLE
Mypy complaines with newest release (20.07), code still works

### ISSUE THREAD

**Issue by LudwigSchallner** [maxbachmann/Levenshtein#43]
Hey,
Me again :D 
So i just pushed some unreleated code changes and the pipeline throws are rather confusing error. My tests and stuff still work totally fine. 
it seems that editops does not support on a type hint level list of str but only str it self. Even tho its still working with list of str like before.

Here is the error:
`error: Argument 1 to "editops"
07:57:22  has incompatible type "Union[str, List[str]]"; expected "str"  [arg-type]
07:57:22              squenence_one,
07:57:22              ^~~~~~~~~~~~~
07:57:22  error: Argument 2 to "editops"
07:57:22  has incompatible type "Union[str, List[str]]"; expected "str"  [arg-type]
07:57:22              squenence_two,
07:57:22              ^~~~~~~~~~~~~`

So with 20.6 its does not throw that error. 
To give more insight of what im doing/using the editops for calculated the word matching rate. So basically just the matching rate based on word with levenshtein. For that I transform the given sentence words into hex. So your code detects those hex numbers as one entity. And so I can calculate the levenhstein distance on a word level. 

So im a bit confused if this is intended behavior/change

**Comment by maxbachmann** [comment]
Previous to `v0.20.7` the type hints did not get installed, which is why this type error did not occur. So this was just never typed correctly.

I agree this should be typed as `Sequence[Hashable]` instead of `str`, since this is supported by `rapidfuzz` which is used for the underlying implementation.

> For that I transform the given sentence words into hex

The implementations supports any sequence of hashable. So e.g. the following would work
```python
>>> import Levenshtein
>>> Levenshtein.editops(["test", "test2"], ["test", "test3"])
[('replace', 1, 1)]
```
as long as you are fine with equality defined as the hash of the corresponding string.

**Comment by maxbachmann** [comment]
This should be fixed in `v0.20.8`

**Comment by LudwigSchallner** [comment]
yes, its fixed :)
